wp_api/src/login/login_client.rs

```diff
@@ -110,14 +110,15 @@ impl WpLoginClient {
             }
         };
         let api_details: WpApiDetails =
-            match serde_json::from_slice::<WpApiDetails>(&fetch_api_details_response.body) {
+            match WpApiDetails::try_from(fetch_api_details_response.body) {
                 Ok(api_details) => api_details,
                 Err(error) => {
+                    // println!("Error parsing API details: {}", String::from_utf8(fetch_api_details_response.body).unwrap());
```

Comment:
```suggestion
```

Reply:
It'd be super handy for debugging if there were a way to fetch the body from the error type